src/service/cli/generate.js

```diff
@@ -9,66 +9,47 @@ const {
   getRandomArrayItem,
   getRandomArrayItems,
 } = require(`../../utils`);
-
-const FILE_NAME = `mock.json`;
-const DEFAULT_COUNT = 1;
-const MAX_COUNT = 1000;
-const PAST_MONTH_LIMIT = 3;
-const DATE_FORMAT = `yyyy-LL-dd HH:mm:ss`;
-
 const {
-  TITLES,
-  ANOUNCES,
-  CATEGORIES,
-} = require(`./text-content`);
-
-const AnounceRestrict = {
-  min: 1,
-  max: 5,
-};
+  CliCommand,
+  MocksConfig,
+  FullTextRestrict,
+} = require(`./constants`);
 
-const FullTextRestrict = {
-  min: 1,
-  max: ANOUNCES.length - 1,
-};
-
-const CategoryRestrict = {
-  min: 1,
-  max: 3,
-};
-
-const getTitle = () => getRandomArrayItem(TITLES);
+const getTitle = () => getRandomArrayItem(MocksConfig.TITLES);
 
 const getDate = () => {
   const now = DateTime.now();
   const past = now.minus({
-    month: PAST_MONTH_LIMIT,
+    month: MocksConfig.PAST_MONTH_LIMIT,
   });
 
   const nowTs = now.valueOf();
   const pastTs = past.valueOf();
   const randomTs = getRandomIntInclusive(nowTs, pastTs);
 
-  return DateTime.fromMillis(randomTs).toFormat(DATE_FORMAT);
+  return DateTime
+    .fromMillis(randomTs)
+    .toFormat(MocksConfig.DATE_FORMAT);
 };
 
-const getSentences = ({min, max}) => {
-  const count = getRandomIntInclusive(min, max);
-
-  return getRandomArrayItems(ANOUNCES, count).join(` `);
+const getSentences = ({MIN, MAX}) => {
+  const count = getRandomIntInclusive(MIN, MAX);
+  return getRandomArrayItems(MocksConfig.ANOUNCES, count).join(` `);
 };
 
-const getAnounce = () => getSentences(AnounceRestrict);
+const getAnounce = () => {
+  return getSentences(MocksConfig.ANOUNCE_RESTRICT);
+};
 
 const getFullText = () => getSentences(FullTextRestrict);
 
 const getCategories = () => {
-  const count = getRandomIntInclusive(CategoryRestrict.min, CategoryRestrict.max);
-  return getRandomArrayItems(CATEGORIES, count);
+  const count = getRandomIntInclusive(MocksConfig.CATEGORY_RESTRICT.MIN, MocksConfig.CATEGORY_RESTRICT.MAX);
+  return getRandomArrayItems(MocksConfig.CATEGORIES, count);
 };
 
 const publicationGenerator = (count) => {
-  return Array(count).fill(``).map(() => {
+  return Array.from(new Array(count), () => {
```

Comment:
```suggestion
  return new Array.from(new Array(count), () => {
```

Reply:
A constructor name should not start with a lowercase. _eslint([new-cap](https://eslint.org/docs/rules/new-cap))_